sphinx_gherkindoc/parsers/behave.py

```diff
@@ -1,11 +1,47 @@
 """Helper functions for writing rST files with behave parser."""
 import behave.parser
 
-from .base import BaseFeature
+from .base import BaseModel
 
 
-class Feature(BaseFeature):
+class BehaveModel(BaseModel):
+    """Custom model for Behave-parsed objects."""
+
+    @property
+    def description(self):
```

Comment:
We should have type hints on all of these things so mypy can make sure we are good. This returns `List[str]`, but that could just as easily be `str`

Reply:
This can be a FF. There's likely some important tweaks with mypy needed since mypy should have caught that we were passing `None` in for scenario in `writer.py` where you changed it.